WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java

```diff
@@ -140,12 +139,21 @@ public void onRefreshStarted() {
 
         setTitle(R.string.stats);
 
+        mOuterScrollView = (ScrollViewExt) findViewById(R.id.scroll_view_stats);
+        mOuterScrollView.setScrollViewListener(this);
+
         if (savedInstanceState != null) {
-            mNavPosition = savedInstanceState.getInt(SAVED_NAV_POSITION);
             mResultCode = savedInstanceState.getInt(SAVED_WP_LOGIN_STATE);
             mLocalBlogID = savedInstanceState.getInt(ARG_LOCAL_TABLE_BLOG_ID);
             mCurrentTimeframe = (StatsTimeframe) savedInstanceState.getSerializable(SAVED_STATS_TIMEFRAME);
             mRequestedDate = savedInstanceState.getString(SAVED_STATS_REQUESTED_DATE);
+            final int[] position = savedInstanceState.getIntArray(SAVED_STATS_SCROLL_POSITION);
```

Comment:
Is it necessary to save/restore both the `x` and `y` positions? Unless the user can scroll horizontally, the `y` position should suffice. Also, we can skip this if the restored position is zero.